### PR TITLE
Misleading Add/Edit Drug Form

### LINKED ISSUE #3114: Misleading Add/Edit Drug Form
### Describe the bug

Misleading Button on add/edit drug form

### To Reproduce
Open add/edit druf form
a. Delete button appears while adding a drug which should not be the case
b. Submit buttom name should be dynamic based on the action

Currently buttons shown below are fixed on both add/edit drug form.
Screenshot from 2020-03-08 03-34-21

### Client configuration

- Browser: Chrome
- Operating system: Linux Ubuntu 18.04

### PR DESCRIPTION
**Pull request by im-Amitto**
Fixes #3114 

#### Short description of what this resolves:
Buttons on the form are generated based on the condition

### CODE REVIEW

File: interface/drugs/add_edit_drug.php [hunk 1]
```diff
@@ -363,8 +363,9 @@ function sel_related() {
     'related_code' => '',
     );
 }
+$title = $drug_id ? "Update Drug" : "Add Drug";
```
Comment (bradymiller) [inline interface/drugs/add_edit_drug.php]: for translations, much better to translate the string so it's picked up in the translation pipeline collection script(s):
https://github.com/openemr/openemr/blob/master/contrib/util/language_translations/README

so would do:
```
$title = $drug_id ? xl("Update Drug") : xl("Add Drug");
```
And then escape them below via text() call

**Comment by bradymiller** [comment]
completed review. just noted a minor issue.

**Comment by im-Amitto** [comment]
@bradymiller Fixed :)

**Comment by bradymiller** [comment]
hi @im-Amitto , Code looks good and bringing into the codebase. Thanks for the improvement! -brady

**Pull request merged**